src/Bugsnag/Notifier.cs

```diff
@@ -39,39 +42,71 @@ public void Send(Event errorEvent)
                 Send(notification);
         }
 
+        public void SendStoredReports()
+        {
+            try
+            {
+                foreach (var storedReport in Store.ReadStoredJson())
+                {
+                    SendJson(storedReport);

```

Comment:
This calls `SendJson` which uses a try-catch and will store the error report in case of a failure. There should be no cascading effect on the other reports. Bugsnag will retry on next app start. Errors should not be discarded